source/whats-new.txt

```diff
@@ -270,19 +270,35 @@ To learn more, see the `v4.12.0 Release Highlights <https://github.com/mongodb/n
 What's New in 4.11
 ------------------
 
+Prioritization Order in Monitoring
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 When connecting to MongoDB Server version 6.0 or later, the driver prioritizes
 ``electionId`` settings before ``setVersion`` settings during Server Discovery and
 Monitoring events. In previous versions, the prioritization order was reversed.
 
+Changes to AWS Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you install the optional ``aws-sdk/credential-providers``
+dependency, the driver uses the AWS SDK to retrieve AWS credentials from the
+environment.
+
+To learn more about this behavior, see the :ref:`MONGODB-AWS
+<mongodb-aws>` section of the Authentication Mechanisms guide.
+
+Mutually Recursive Schema Type Checking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 This release also includes added support for **mutually
```

Comment:
Suggestion

Given how you've restructured this beginning, the "also" here seems out of place and that it doesn't flow well.

```suggestion
This release includes added support for **mutually
```